pkg/statistics/handle/autoanalyze/autoanalyze.go

```diff
@@ -116,6 +117,7 @@ func (sa *statsAnalyze) FinishAnalyzeJob(job *statistics.AnalyzeJob, failReason
 
 // DeleteAnalyzeJobs deletes the analyze jobs whose update time is earlier than updateTime.
 func (sa *statsAnalyze) DeleteAnalyzeJobs(updateTime time.Time) error {
+	defer util.Recover(metrics.LabelAnalyze, "DeleteAnalyzeJobs", nil, false)
```

Comment:
can we wrap it into CallWithSCtx as a whole

Reply:
Sure, I have updated it.